test/unit/test/tmp_filename_test.cpp

```diff
@@ -34,8 +34,10 @@
 
 #include <gtest/gtest.h>
 #include <fstream>
-#include "tmp_filename.hpp"
 
+#include <seqan3/test/tmp_filename.hpp>
+
+namespace fs = seqan3::filesystem;
```

Comment:
why was this added here?

Reply:
the test case didn't work and I fixed it.

The test case was originally in `/test/include/seqan3/test/tmp_filename_test.cpp` (in the header file folder for test stuff) where it was never build, executed and tested.

I moved it to `/test/unit/test/tmp_filename_test.cpp` and all of the sudden this never tested file needed some changes.